data/menu_data.rb

```diff
@@ -0,0 +1,9 @@
+MENU = {
```

Comment:
Smart! Just putting the data in a ruby file and referencing it without creating a class which would have been more challenging to manage